content/en/docs/devops-user-guide/how-to-integrate/sonarqube.md

```diff
@@ -143,12 +143,12 @@ To integrate SonarQube into your pipeline, you must install SonarQube Server fir
 
    ![webhook-page-info](/images/docs/devops-user-guide/tool-integration/integrate-sonarqube-into-pipeline/webhook-page-info.jpg)
 
-### Step 4: Add the SonarQube configuration to ks-installer
+### Step 4: Add the SonarQube configuration to ks-devops
```

Comment:
If `devops-config` is to be edited in the following steps, I think we should change `ks-devops` to `devops-config`.

Reply:
See https://github.com/kubesphere/website/pull/2152/commits/aa4381a4b037e96bb37ee5834c1b16dea15203d1